app/src/main/java/zegreanu/cristi/geoquiz/QuizActivity.kt

```diff
@@ -74,6 +83,8 @@ class QuizActivity : AppCompatActivity() {
         super.onSaveInstanceState(outState)
         Log.i(TAG, "onSaveInstanceState")
         outState.putInt(KEY_INDEX, currentIndex)
```

Comment:
We can probably use apply here:
outState.apply {
    putInt(...)
    putInt(...)
....
}